webui/Makefile

```diff
@@ -41,7 +47,8 @@ build-react:
 build-elm:
```

Comment:
At this point, we may as well go ahead and use the `build-%` pattern for `build-elm` and `build-react` too.